users/models.py

```diff
@@ -107,6 +107,16 @@ def find_random_unused_phone_number(self) -> str:
 
 
 class JustfixUser(AbstractUser):
+    class Meta:
+        verbose_name = "user"
+        verbose_name_plural = "users"
```

Comment:
These two need to be preserved from `AbstractBaseUser` or else the model will be called "justfix user" instead of "user" (which isn't a big deal really, but it's not the point of this particular change, so we want to preserve the original name).